pkg/adapter/v2/cloudevents.go

```diff
@@ -67,6 +79,30 @@ func NewCloudEventsClientCRStatus(target string, ceOverrides *duckv1.CloudEventO
 	}, nil
 }
 
+func withTimeOut(duration time.Duration) http.Option {
+	return func(p *http.Protocol) error {
+		if p == nil {
+			return fmt.Errorf("http target option can not set nil protocol")
+		}
+		if p.Client == nil {
+			p.Client = &nethttp.Client{}
+		}
+		p.Client.Timeout = duration
+		return nil
+	}
+}
+
+func getSinkWaitDuration() time.Duration {
```

Comment:
Can't you use `GetSinkWaitDuration` made before?

Reply:
No introduces cycle

Reply:
If you put in v2/config.go, it doesn't